packages/terra-site/src/examples/card/Index.jsx

```diff
@@ -26,8 +27,10 @@ const CardExamples = () => (
     <CardPadding />
     <h2 id="CardPaddingVertical">Or just vertical</h2>
     <CardPaddingVertical />
-    <h2 id="CardPaddingHorizontal">or just horizontal</h2>
+    <h2 id="CardPaddingHorizontal">Or just horizontal</h2>
     <CardPaddingHorizontal />
+    <h2 id="CardPaddingHR">We can have a Horizontal line ignore the padding</h2>
```

Comment:
This seems like an inaccurate description. Since horizontal rule isn't ignoring padding, it's just not getting padding added. Perhaps reword to something like: 

> Selectively add padding to child elements